### ISSUE TITLE
[BUG-REPORT] 'DataFrameLocal' object has no attribute 'to_records'

### ISSUE THREAD

**Issue by pedrobzz** [vaexio/vaex#1515]
So, i was reading the [vaex docs](https://vaex.io/docs/example_io.html#Expression-representations) and i've found in the examples the following code:
```
df = vaex.open('./data/io/sample_simple.hdf5')
records = df.to_records()
```
This is exactly what i need but when i execute that i have the following error:
`AttributeError: 'DataFrameLocal' object has no attribute 'to_records'`

How do i fix that? On the API i've found that `DataFrameLocal` really doesn't have this method, but on there it isn't documented `to_dict`, and that works fine for me.

My Code ([screenshot](https://i.imgur.com/zjle2bC.png)):
```
df = vx.open('../data/hdf5/empresas.hdf5')
df.to_records()
```

**Comment by maartenbreddels** [comment]
We *just* released this, vaex-core==4.4.0 :)

**Comment by pedrobzz** [comment]
> We _just_ released this, vaex-core==4.4.0 :)

Oh ok, it works, thx! Sometimes its a bit slower than normal but its way better than my last implementation! Thanks for the great work, i'm in love with Vaex

**Comment by maartenbreddels** [comment]
Thanks for letting us know :)
